src/connectors/relaxfm.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+Connector.playerSelector = '.main-player__wrapp';
+
+Connector.artistSelector = '.TitleExecutorAir';
+
+Connector.trackSelector = '.TitleTracksAir';
+
+Connector.isPlaying = () => $('.toggleplayer').hasClass('stop');
```

Comment:
Could be simplified to this.

```suggestion
Connector.pauseButtonSelector = '.toggleplayer.stop';
```

Would avoid the dependency on jQuery which is being phased out in preparation for v3 with bootstrap 5